package.json

```diff
@@ -2,22 +2,28 @@
   "name": "idtoken-verifier",
   "version": "1.2.0",
   "description": "A lightweight library to decode and verify RS JWT meant for the browser.",
-  "main": "src/index.js",
+  "main": "build/idtoken-verifier.cjs.js",
+  "umd:main": "build/idtoken-verifier.umd.js",
+  "module": "build/idtoken-verifier.esm.js",
+  "source": "src/index.js",
   "scripts": {
     "start": "gulp dev",
-    "build": "gulp build",
-    "test": "mocha test/**/*.test.js",
-    "test:watch": "mocha --watch  -R min test/*.test.js",
-    "test:coverage": "istanbul cover _mocha -R test/*.test.js",
-    "ci:test": "istanbul cover _mocha --report lcovonly -R test/** -- -R mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
+    "build": "microbundle --target web --external none",
+    "test": "NODE_ENV=test mocha --require @babel/register test/**/*.test.js --exit",
```

Comment:
Use cross-env here for compatibility with Windows?

Reply:
yessssssss. pushed this change 🎉